db/config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   host: 'localhost',
-  user: 'root',
+  username: 'root',
```

Comment:
Perhaps consider adding this file to your .gitignore?
Or otherwise obscuring the username/password for your db?
Guess it's not a huge issue for this situation though (since your computer isn't connected to the internet 24-7 and the IP isn't publicly available). 🤷‍♀️

Reply:
That is a good point. Ideally if this was sensitive information I would either .gitignore it obscure it like you said. I will keep that in mind for future projects where it critical. Not sure how I would do that in this case and still have the project work without hard coding it. Suggestions?

Reply:
Easiest way would probably be through a environment config file somewhere, but for now this is fine